proposals/lip-0048.md

```diff
@@ -25,15 +25,57 @@ This LIP defines the fee system in a modular way, as currently [used in the Lisk
 
 ## Rationale
 
-### Fee Token ID
+Fees for blockchain transactions are necessary for a variety of reasons, including state growth, node computation, and block space utilization.
 
-Each chain can configure the token used to pay fees. On the Lisk mainchain, the token used for transaction fees is the LSK token.
+The current implementation lacks an automated method for calculating fees in all of these cases. Instead, we automate the calculation of block space fees using a *minimum fee* approach, similar to Bitcoin's mechanism, and allow developers discretion to use the [`payFee`](#payfee) method for fee assignment in other situations, e.g., when transaction implies state growth or more significant computational effort required by a node.
```

Comment:
Just to clarify: This is similar to Bitcoin in the sense that the only relevant metric for the calculation of the min fee is the transaction size, but bitcoin does not have a min fee (that I know of). For simplicity I would drop the 'similar to Bitcoin's mechanism' sentence.

Reply:
Also, I would not start the paragraph by saying that we lack some feature. We could just outline the two main topics (minimum fee based on blockspace) and extra command fees and say that they are explained below.

Reply:
```suggestion
The fee system proposed here, automates the calculation of block space fees using a *minimum fee* approach: the transaction fee should exceed a threshold which depends on the size of the transaction, as we explain below. Fees related to other reasons (e.g., state growth, computational effort) are determined by the corresponding parts of the protocol (e.g., extra fee to register a validator in the PoS module); the fee module takes care of the handling of such fees, via the [`payFee`](#payfee) method.
```

Reply:
Thanks for the suggestion, I changed the parahraph as suggested.